backend/api/internal/handlers/floorplans/seats/handlers.go

```diff
@@ -0,0 +1,97 @@
+package seats
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+
+	"github.com/lanets/floorplanets/backend/api/internal/handlers/decorators"
+	"github.com/lanets/floorplanets/backend/app"
+	"github.com/lanets/floorplanets/backend/models"
+)
+
+func seatsGetHandler(app *app.App) http.Handler {
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		floorplan := decorators.FloorplanFromContext(r.Context())
+
+		err := app.LoadFloorplanSeats(floorplan)
+
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+
+		fmt.Fprint(w, models.SeatListtoJson(floorplan.Seats))
+
+	}
+
+	handler = decorators.FloorplanContext(app, handler)
+	handler = decorators.JsonHeaders(handler)
+
+	return http.HandlerFunc(handler)
+}
+
+func seatsPostHandler(app *app.App) http.Handler {
```

Comment:
I think we should think about the use case for this call. Don't you think we should have an endpoint to add a list of Seats ? 

IMO the common use case is going to be the addition of a "row" of seats. Now, the "row" itself is abstract and could be something else, but most likely the addition of seats in a floorplan is going to be something else than a single call by seat. 

A floorplan of 3000 seats is a lot of API calls. I think this endpoint should exist, just so that we're following the rest format. But an other endpoint is going to be needed in the future.

TLDR: Jpense qu'on va avoir besoin d'un call qui permet d'ajouter une liste de seats.